### PR TITLE
move rsync flag --compress to ssh section. It will be used only if de…

### PR DESCRIPTION
**Pull request by zwotzie**
…stination is accessed via ssh

As discussed in the issue #142.

I had thought about an extra option, but this would have affected backwards compatibility.

Best Regards
 Ralf Hense

### CODE REVIEW

File: rsync_tmbackup.sh [hunk 1]
```diff
@@ -463,6 +464,7 @@ while : ; do
 
 	CMD="rsync"
 	if [ -n "$SSH_CMD" ]; then
+	    $RSYNC_FLAGS="$RSYNC_FLAGS --compress"
```
Comment (laurent22) [inline rsync_tmbackup.sh]: Please use tabs for indentation

**Comment by laurent22** [comment]
Thanks, all looking good other than the indentation issue.

**Comment by zwotzie** [comment]
here you are ;)

**Comment by laurent22** [comment]
Looks good now, thanks!

**Comment by cr1901** [comment]
When running on my machine, I get something like (ignore the line number):

```
/home/wjones/bin/rsync_tmbackup.sh: line 557: -D --numeric-ids --links --hard-l$
nks --one-file-system --itemize-changes --times --recursive --perms --owner --g$oup --stats --human-readable --dry-run=-D --numeric-ids --links --hard-links --$
ne-file-system --itemize-changes --times --recursive --perms --owner --group --$
tats --human-readable --dry-run --compress: command not found
```

Should be changed to be `RSYNC_FLAGS="$RSYNC_FLAGS --compress"`?

**Pull request merged**